charts/ocis/templates/postprocessing/deployment.yaml

```diff
@@ -0,0 +1,111 @@
+{{- $_ := set . "appName" "postprocessing" -}}
+{{- $_ := set . "resources" (default (default (dict) .Values.resources) .Values.services.postprocessing.resources) -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .appName }}
+  namespace: {{ template "ocis.namespace" . }}
+  labels:
+    {{- include "ocis.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .appName }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ .appName }}
+        {{- include "ocis.labels" . | nindent 8 }}
+    spec:
+      securityContext:
+          fsGroup: {{ .Values.securityContext.fsGroup }}
+          fsGroupChangePolicy: {{ .Values.securityContext.fsGroupChangePolicy }}
+      {{- with .Values.topologySpreadConstraints }}
+      topologySpreadConstraints:
+        {{- tpl . $ | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ .appName }}
+          image: {{ template "ocis.image" $ }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["ocis"]
+          args: ["postprocessing", "server"]
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: {{ .Values.securityContext.runAsUser }}
+            runAsGroup: {{ .Values.securityContext.runAsGroup }}
+            readOnlyRootFilesystem: true
+          env:
+            - name: MICRO_REGISTRY
+              value: kubernetes
+
+            - name: POSTPROCESSING_LOG_COLOR
+              value: {{ .Values.logging.color | quote }}
+            - name: POSTPROCESSING_LOG_LEVEL
+              value: {{ .Values.logging.level | quote }}
+            - name: POSTPROCESSING_LOG_PRETTY
+              value: {{ .Values.logging.pretty | quote }}
+
+            # - name: POSTPROCESSING_DEBUG_PPROF
+            #   value: {{ .Values.debug.profiling | quote }}
+
+            # - name: POSTPROCESSING_DEBUG_ADDR
+            #   value: 0.0.0.0:TODO
+
+            - name: POSTPROCESSING_VIRUSSCAN
+              value: "false" # TODO: configurable when antivirus service is added
```

Comment:
Not critical because it is set to false. But `POSTPROCESSING_VIRUSSCAN` is deprecated. Instead use `POSTPROCESSING_STEPS=virusscan` (it can support additional steps like `POSTPROCESSING_STEPS=virusscan,delay` hence the change)